Makefile

```
@@ -0,0 +1,19 @@
+.PHONY: vet lint test
+
+VETTOOL := $(shell which statictest)
+TEST_RUN=\^TestIteration1\$$
+SOURCE_PATH=.
```

```suggestion
SOURCE_PATH ?= ${CURDIR}
```

https://www.gnu.org/software/make/manual/html_node/Quick-Reference.html